### PR TITLE
Fixed crash caused by lvcod64.dll on AVI settings popup

### PR DESCRIPTION
**Pull request by turtletooth**
lvcod64.dll was causing a crash on using the AVI settings popup.  This is the same problem from #52.  I used the same solution as in #52.

### CODE REVIEW

**Comment by turtletooth** [comment]
What needs to be changed for this to pass?  Works on Windows 10 64 bit.

File: toonz/sources/toonzlib/avicodecrestrictions.cpp [hunk 1]
```diff
@@ -220,20 +220,34 @@ QMap<std::wstring, bool> AviCodecRestrictions::getUsableCodecs(
...
+          hic = ICOpen(icinfo.fccType, icinfo.fccHandler, ICMODE_COMPRESS);
+        } __except (EXCEPTION_EXECUTE_HANDLER) {
+        }
+      }();
+#else
+      try {
+        hic = ICOpen(icinfo.fccType, icinfo.fccHandler, ICMODE_QUERY);
+        ICMODE_COMPRESS
+        catch (...) {
+        }
```
Comment (shun-iwasawa) [inline toonz/sources/toonzlib/avicodecrestrictions.cpp]: Please fix here in order to enable to build when _MSC_VER is not defined.

**Comment by shun-iwasawa** [comment]
I'm not actually understand the issue modified here as I couldn't reproduce the crash on using the AVI settings popup. Is there any condition to reproduce the problem?

**Comment by turtletooth** [comment]
I don't know if it is a Windows 10 issue, but the crash is consistent for me. 
avi_crash

> Please fix here in order to enable to build when _MSC_VER is not defined.

I will look into that.

**Comment by shun-iwasawa** [comment]
Can one of the admins verify this patch?

**Comment by RodneyBaker** [comment]
REF also:  https://github.com/opentoonz/opentoonz/issues/46

**Pull request closed without merging**